source/utilities.tex

```diff
@@ -17646,7 +17646,7 @@
 \tcode{template<class T, class U>}\br
  \tcode{struct is_layout_compatible;}                 &
  \tcode{T} and \tcode{U} are layout-compatible\iref{basic.types}    &
- \tcode{T} and \tcode{U} are complete types,
+ \tcode{T} and \tcode{U} shall be complete types,
```

Comment:
This is not an editorial change.

Reply:
I believe it is: It is stating requirements on the incoming T and U from the user. Turning that into "shall" makes the intent more obvious, but doesn't change the intent.

Reply:
The wording in the paper was wrong, the author misunderstood LWG guidance. I corrected it (restoring consistency with the local wording, and with LWG conventions).